.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,39 @@
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: #enhancement
+body:
+  - type: textarea
+    id: formulation
+    attributes:
+      label: Is your feature request related to a problem? Please describe
+      description: A clear and concise description of what the problem is
+      placeholder: |
+        I'm always frustrated when [...]
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
```

Comment:
Add extra `textarea` about how it affected the development. Like, what was the negative impact?

Reply:
@PeyTy done too